ff-server.yaml

```diff
@@ -0,0 +1,242 @@
+# Food Finder App
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ff-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ff-app
+  template:
+    metadata:
+      labels:
+        app: ff-app
+    spec:
+      containers:
+        - name: ff-finder-service
+          image: gcr.io/cal-intern-project/nholbrook-ff-finder-service:latest
+          ports:
+            - containerPort: 50051
+              protocol: TCP
+        - name: ff-supplier-service
+          image: gcr.io/cal-intern-project/nholbrook-ff-supplier-service:latest
+          ports:
+            - containerPort: 50052
+              protocol: TCP
+        # - name: endpoint-service
+        #   image: gcr.io/endpoints-release/endpoints-runtime:1
+        #   args:
+        #     - --http2_port=9000
+        #     - --backend=grpc://127.0.0.1:50051
+        #     - --service=helloworld.endpoints.cal-intern-project.cloud.goog
+        #   ports:
+        #     - containerPort: 9000
+        #       protocol: TCP
+---
+# Food Finder Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: ff-finder-service
+spec:
+  ports:
+  - protocol: TCP
+    port: 50051
+    name: ff-port
+  - protocol: TCP
+    port: 50052
+    name: supplier-port
+  selector:
+    app: ff-app
+  type: LoadBalancer
+---
+# Vendor 1 Config Map (Meijer)
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vendor1-configmap
+data:
+  inventory.txt: \
+    milk 2.19 5
+    eggs 2.38 8
+    bread 1.99 10
+    pepper 1.50 18
+    butter 3.29 4
+---
+# Vendor 1 App (Aldi)
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ff-vendor1-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ff-vendor1-app
+  template:
+    metadata:
+      labels:
+        app: ff-vendor1-app
+    spec:
+      containers:
+        - name: ff-vendor1-service
+          image: gcr.io/cal-intern-project/nholbrook-ff-vendor-service:latest
+          ports:
+            - containerPort: 50060
+          env:
+            - name: VENDOR_NAME
+              value: Meijer
+            - name: VENDOR_ADDRESS
+              value: 34.75.101.194:50060
```

Comment:
where does this address come from?

Reply:
I was hardcoding them, but I've updated it to fetch the current IP using kubectl.